modules/services/polkit-gnome.nix

```diff
@@ -0,0 +1,35 @@
+{
+  lib,
+  pkgs,
+  config,
+  ...
+}: let
+  inherit (lib) mkIf mkEnableOption mkOption mkPackageOption;
+  cfg = config.services.polkit-gnome;
+in {
+  meta.maintainers = []; # TODO: add maintainer
```

Comment:
Please add maintainer. Ideally yourself. Feel free to add me as well or instead if you'd prefer not to add yourself.